packages/ui-templates/lib/render.ts

```diff
@@ -82,19 +82,19 @@ export const RenderPlugin = () => {
         // Serialize into a js function
         const chunks = html.split(/\{{2,3}[^{}]+\}{2,3}/g).map(chunk => JSON.stringify(chunk))
         const hasMessages = chunks.length > 1
-        let templateString = chunks.shift()
+        const templateString = [chunks.shift()]
```

Comment:
I'm not sure this is a more performant approach... Let's skip this particular change for now.

Reply:
Sure thing! Generally speaking pushing to an array has a better space complexity than concataneting strings because each concatenation creates a new string instance, whereas pushing to an array uses the same instance, so the memory usage in this specific execution, especially if templateString is iterate over a lot, would be far lower for the array method than the string method. But sure, it can be kept as is until the more impactful areas are taken care of

Reply:
Understood. TIL. However this is build time code (while bundling up templates) and never impacts a user. I am happy to keep it as is.

Reply:
Sounds good to me!